frontend/packages/knative-plugin/src/components/revisions/RevisionsPage.tsx

```diff
@@ -0,0 +1,16 @@
+import * as React from 'react';
+import { ListPage } from '@console/internal/components/factory';
+import { referenceForModel } from '@console/internal/module/k8s';
+import RevisionList from './RevisionList';
+import { RevisionModel } from '../../models';
+
+const RevisionsPage: React.FC = (props) => (
```

Comment:
Can you create an `interface` for the props here as well?

Reply:
Added props to get namespace from react-router match params.